torch/csrc/jit/api/module.h

```diff
@@ -213,19 +213,23 @@ struct TORCH_API Module : public Object {
 
   void save(
       std::ostream& out,
-      const ExtraFilesMap& extra_files = ExtraFilesMap()) const;
+      const ExtraFilesMap& extra_files = ExtraFilesMap(),
+      bool save_debug_info = false) const;
 
   void save(
       const std::string& filename,
-      const ExtraFilesMap& extra_files = ExtraFilesMap()) const;
+      const ExtraFilesMap& extra_files = ExtraFilesMap(),
+      bool save_debug_info = false) const;
```

Comment:
We should not add this to anything other than `_save_for_mobile` as that is what we are doing. Please revert these ones.

Reply:
Thanks for catching it! I'll fix that.